examples/gno.land/r/stefann/home/home.gno

```diff
@@ -0,0 +1,274 @@
+
+	out += "</div>\n\n"
+
+	out += "<div>\n\n"
+	out += aboutMe[0] + "\n\n"
+	out += "</div>\n\n"
+
+	out += "<div>\n\n"
+	out += aboutMe[1] + "\n\n"
+	out += "</div>\n\n"
+
+	out += "</div><!-- /rows-3 -->\n\n"
+
+	return out
+}
+
+func renderTips() string {
+	out := `<div class="jumbotron" style="display: flex; flex-direction: column; justify-content: flex-start; align-items: center; padding-top: 40px; padding-bottom: 50px; text-align: center;">` + "\n\n"
+
+	out += `<div class="rows-2" style="max-width: 500px; width: 100%; display: flex; flex-direction: column; justify-content: center; align-items: center;">` + "\n"
+
+	out += `<h1 style="margin-bottom: 50px;">Help Me Travel The World</h1>` + "\n\n"
+
+	out += renderTipsJar() + "\n"
+
+	out += ufmt.Sprintf(`<strong style="font-size: 1.2em;">I am currently in %s, <br> tip the jar to send me somewhere else!</strong>`, cities[currentCityIndex].Name)
+
+	out += `<br><span style="font-size: 1.2em; font-style: italic; margin-top: 10px; display: inline-block;">Click the jar, tip in GNOT coins, and watch my background change as I head to a new adventure!</span></p>` + "\n\n"
+
+	out += renderSponsors()
+
+	out += `</div><!-- /rows-2 -->` + "\n\n"
+
+	out += `</div><!-- /jumbotron -->` + "\n"
+
+	return out
+}
+
+func formatAddress(address string) string {
+	if len(address) <= 8 {
+		return address
+	}
+	return address[:4] + "..." + address[len(address)-4:]
+}
+
+func renderSponsors() string {
+	out := `<h3 style="margin-top: 5px; margin-bottom: 20px">Sponsor Leaderboard</h3>` + "\n"
+
+	if len(sponsors) == 0 {
+		out += `<p style="text-align: center;">No sponsors yet. Be the first to tip the jar!</p>` + "\n"
+	} else {
+		numSponsors := len(sponsors)
+		if numSponsors > maxSponsors {
+			numSponsors = maxSponsors
+		}
+
+		out += `<ul style="list-style-type: none; padding: 0; border: 1px solid #ddd; border-radius: 8px; width: 100%; max-width: 300px; margin: 0 auto;">` + "\n"
+
+		for i := 0; i < numSponsors; i++ {
+			sponsor := sponsors[i]
+			isLastItem := (i == numSponsors-1)
+
+			padding := "10px 5px"
+			border := "border-bottom: 1px solid #ddd;"
+
+			if isLastItem {
+				padding = "8px 5px"
+				border = ""
+			}
+
+			out += ufmt.Sprintf(
+				`<li style="padding: %s; %s text-align: left;">
+                    <strong style="padding-left: 5px;">%d. %s</strong> 
+                    <span style="float: right; padding-right: 5px;">%s</span>
+                </li>`,
+				padding, border, i+1, formatAddress(sponsor.Address.String()), sponsor.Amount.String(),
+			)
+		}
+
+	}
+
+	return out
+}
+
+func renderTipsJar() string {
+	out := ufmt.Sprintf(`<a href="%s" target="_blank" style="display: block; text-decoration: none;">`, jarLink) + "\n"
+
+	out += `<img src="https://i.ibb.co/4TH9zbw/tips-jar.png" alt="Tips Jar" style="width: 300px; height: auto; display: block; margin: 0 auto;">` + "\n"
+
+	out += `</a>` + "\n"
+
+	return out
+}
```

Comment:
I explained the purpose of  `jarLink` and why it can't be a constant in the comment below. https://github.com/gnolang/gno/pull/2918#discussion_r1796891994

Or did you mean to simplify the string concatenation by building the entire string in one call to ufmt.Sprintf? It's done this way for clarity, but I can change it if needed.

Reply:
Ah, I missed you utilize a change-able var here 🤦‍♂️